src/main/java/mate/academy/springboot/datajpa/dto/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+public interface DtoMapper<M, Q, S> {
```

Comment:
It makes sense

Reply:
I mean it can have it, no troubles with that, but most of the time there is no need for that. 
So in your case that purely depends on how you want to do this task